deployment/run_server

```diff
@@ -2,9 +2,16 @@
 
 fc-cache -v /usr/share/fonts/user
 
+BUILD_DIR=/var/www/image-exporter/build
+if [ ! -z ${PLOTLY_JS_SRC+x} ]; then
```

Comment:
I'm unclear on the purpose of `+x`. Can you please explain?

Reply:
https://stackoverflow.com/a/13864829

Reply:
There shouldn’t be a case when the variable is set to blank string, but this is more robust.. !!

Reply:
OK, but in this case if `PLOTLY_JS_SRC` is set tho the empty string we also want to skip the download, so the simpler approach is preferable:

```bash
if [[ -n "$PLOTLY_JS_SRC" ]]; then
    ....
fi
```

Note that I'm also using `[[` for reasons outlined in https://github.com/progrium/bashstyle

Reply:
> but in this case if PLOTLY_JS_SRC is set tho the empty string we also want to skip the download,

That makes sense to me.. updated .. !!